src/vse_sync_pp/analyzers/analyzer.py

```diff
@@ -260,6 +260,8 @@ def test(self, data):
             return (False, "short test duration")
         if len(data) - 1 < self._duration_min:
             return (False, "short test samples")
+        if len(data.timestamp.diff().astype(float).round(0).tail(-1).unique()) > 1:
```

Comment:
I'm guessing `.astype(int)` will truncate instead of rounding

Reply:
your guess is correct. @nocturnalastro I also added the condition in TIE Analyzer base. Report looking good.